### PR TITLE
Nit Fixes from PR #43 Comment thread

### PR DESCRIPTION
**Pull request by 0tuedon**

### CODE REVIEW

File: tailwind.config.ts [hunk 1]
```diff
@@ -31,6 +31,7 @@ const config: Config = {
           "custom-1200": "#CCCCCC",
           "custom-1300": "#E6E6E6",
           "custom-1400": "#ACACAC",
+          "custom-1500": "#CCCCCC"
```
Comment (Emmanuel-Develops) [inline tailwind.config.ts:34]: the color is defined in `"custom-1200": "#CCCCCC",`

**Review by Emmanuel-Develops — CHANGES_REQUESTED** [review]

**Comment by Emmanuel-Develops** [comment]
https://github.com/bitcointranscripts/registry/pull/43#discussion_r1795617339

[sources page is server rendered](https://github.com/bitcointranscripts/registry/pull/41/files#diff-c7396d8de48474b66ecc796f876d4500ae535f8b15a3976b8929677f1b9e4992) and it doesn't have such issues. Kindly switch to server render and investigate if issue still persists

cc: @0tuedon

File: tsconfig.json [hunk 2]
```diff
@@ -32,5 +34,5 @@
     ".contentlayer/generated",
     ".contentlayer/generated/**/*.json"
   ],
-  "exclude": ["node_modules", ".contentlayer"]
+  "exclude": ["node_modules", ".contentlayer", "public/tag-data.json"]
```
Comment (Emmanuel-Develops) [inline tsconfig.json:37]: any specific reason for excluding `"public/tag-data.json"`? 
we might as well exclude the whole directory `public`

Comment (0tuedon) [reply tsconfig.json:37]: Oh, yh

**Review by Emmanuel-Develops — APPROVED** [review]

**Pull request merged**